express-101/src/index.js

```diff
@@ -1,9 +1,16 @@
 const express = require('express');
 
 const app = express();
+const port = 3000;
 
-// 1) Create Routes
+app.get('/', (req, res) => {
+  res.send('Hey, I am server response');
+});
 
-// 2) Start server on port 3000
+app.get('/movie/:title', (req, res) => {
+  res.send(`My favorite movie is ${req.params.title}`);
```

Comment:
Won't the use of destructuring here make further modifications easy?
Like so: const { title } = req.params;
You won't have to rewrite req.params.title everytime.